clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/IngestionNotificationDispatcher.java

```diff
@@ -72,6 +73,10 @@ private void report(
       try {
         function.apply(notifier);
       } catch (Exception ex) {
+        if (notifier instanceof PartitionPushStatusNotifier) {
```

Comment:
The ideal solution should be letting Controller look at CV instead of `OfflinePushStatus` as `CV` is being used as the SoT for routing.
I wonder how hard it is to switch to CV in Controller for batch push.
Also this fix is a little error-prone as the user can attach the notifiers in a different order and if we decide to go with this approach, I think we need to consolidate offline push status notifier and CV notifier into one wrapper class and always make sure CV error won't mark offline push status as completed.